src/chainparams.cpp

```diff
@@ -189,22 +193,26 @@ class CTestNetParams : public CChainParams {
         consensus.BIP66Height = 330776; // 000000002104c8c45e99a8853285a3b592602a3ccde2b832481da85e9e4ba182
         consensus.CSVHeight = 770112; // 00000000025e930139bac5c6c31a403776da130831ab85be56578f3fa75369bb
         consensus.SegwitHeight = 834624; // 00000000002b980fcd729daaa248fd9316a5200e9b367f4ff2c42453e84201ca
-        consensus.MinBIP9WarningHeight = 836640; // segwit activation height + miner confirmation window
+        consensus.MinVBitsWarningHeight = 836640; // segwit activation height + miner confirmation window
```

Comment:
We had discussed delaying the member/variable renames until after the actual logic changes for better backport/merge/review, and I thought that was a good idea. Are you planning to take that out of the PR? Been waiting to review pending on that...

Reply:
Oh yeah, forgot to do that. I've removed the rename commit that I added in this PR, but #21380 also has a rename commit and it's up to @ajtowns to remove that if desired.